cnap/core/qat.py

```diff
@@ -0,0 +1,231 @@
+"""A QAT module.
```

Comment:
And do we need add the method or machnism to ensure that all session will be released (__del__), or just called by users (eg, close session). What do you think about it.

Reply:
Fix all. Thanks!